src/components/entry/EntryDetails.js

```diff
@@ -1,19 +1,207 @@
+import { Flag, Map, Star } from '@styled-icons/boxicons-solid'
+import React, { useEffect, useState } from 'react'
 import { useRouteMatch } from 'react-router-dom'
+import styled from 'styled-components'
 
-const EntryDetails = () => {
+import { getLocationById, getTypeById } from '../../utils/api'
+import Button from '../ui/Button'
+import { theme } from '../ui/GlobalStyle'
+import IconButton from '../ui/IconButton'
+import { Tag, TagList } from '../ui/Tag'
+import { RESOURCES } from './resources'
+
+const ACCESS_TYPE = {
+  0: "On lister's property",
+  1: 'Received permission from owner',
+  2: 'Public property',
+  3: 'Private but overhanging',
+  4: 'Private property',
+}
+
+const formatISOString = (dateString) =>
```

Comment:
A comment here describing how to interface with this function (return types on error, etc) would be useful